ml_models/use_model.py

```diff
@@ -63,10 +69,29 @@ def get_answer(query):
     ans_list = [get_text_by_ind(ind) for k, ind in indexer.return_closest(query, k=4)]
     return ans_list
 
+
 def get_answer_ind(query):
     ind_list = [get_new_ind_by_ind(ind) for k, ind in indexer.return_closest(query, k=4)]
     return ind_list
 
+
+def get_slack_answer_ind(query, k=4):
+    import requests
+
+    url = f"https://slack.com/api/search.messages?count={k}&query={query}&pretty=1"
+
+    payload = {}
+    headers = {
+        'Authorization': f'Bearer {os.environ["USER_TOKEN"]}'
+    }
+
+    response = requests.request("GET", url, headers=headers, data=payload)
+    matches = json.loads(response.text).get('messages').get('matches')
```

Comment:
What if you will use simple getter `['a']` instead of `.get`? Will there be errors?

Reply:
I think it would be safer to use .get, because site can probably return unexpected values.

Reply:
I mean,`.get` return `None` by default, not `{}`, and next `.get` will not exist.
To avoid it you can use `.get(x, {})`, at least there will be dict methods.
But it's minor comment, you can save it as is.